lib/GetStream/StreamChat/Client.php

```diff
@@ -447,6 +447,21 @@ public function getMessage($messageId)
         return $this->get("messages/" . $messageId);
     }
 
+    /**
+     * @param  array $filterConditions
+     * @param  array $options
+     * @return mixed
+     * @throws StreamException
+     */
+    public function queryMessageFlags($filterConditions, $options=null)
```

Comment:
Looks like that would work, but this pattern is repeated all over the code-base, it might be idiomatic PHP?

For Python it is like this, where you wouldn't use mutable datatypes as default values

Reply:
That's fine for PHP.

Reply:
You merged too quickly @ferhatelmas, we didn't remove the `if` block 😄

Reply:
Nope, will do a pass for whole SDK for the pattern.